cmd/node/main.go

```diff
@@ -1034,8 +1040,18 @@ func loadEconomicsConfig(filepath string) (*config.ConfigEconomics, error) {
 	return cfg, nil
 }
 
-func loadPreferencesConfig(filepath string) (*config.ConfigPreferences, error) {
-	cfg := &config.ConfigPreferences{}
+func loadPreferencesConfig(filepath string) (*config.Preferences, error) {
```

Comment:
you should move all that functions from  main.go in a new file maybe a common file or something like that

Reply:
will leave it like this for now. maybe in a future PR

Reply:
why you cant do this now because you have just to copy a part of the code from main.go in a new file ?